packages/open-next/src/build/createServerBundle.ts

```diff
@@ -280,6 +297,17 @@ function shouldGenerateDockerfile(options: FunctionOptions) {
   return options.override?.generateDockerfile ?? false;
 }
 
+function addDenoJson(outputPath: string, packagePath: string) {
```

Comment:
You said in your PR that it might not be necessary with deno 2, we should add a comment about this in the code as well